app/containers/ITunesProvider/TrackDetailsContainer/index.js

```diff
@@ -86,16 +86,17 @@ export function TrackDetailsContainer({
     <If condition={!loading && trackDetails} otherwise={renderErrorCard()}>
       <Container maxWidth={maxWidth}>
         <Card
```

Comment:
so typically you would move this to another component so that it gets easier to test that component individually. Then you import the new component here and just test whether that component is getting rendered